src/main/resources/templates/db_scripts/create_failed_ledger_rollover_po_line_table.sql

```diff
@@ -0,0 +1,13 @@
+-- Creates table to store failures during rollover
+CREATE TABLE IF NOT EXISTS failed_ledger_rollover_po_line
+(
+  id uuid NOT NULL PRIMARY KEY,
+  rollover_id uuid,
```

Comment:
Please add b-tree index to rollover_id column. It is needed in case we want to filter by rolloverId dirrectly in DB or maybe provide some endpoint to show errors etc in future